cluster/gce/util.sh

```diff
@@ -2132,11 +2132,13 @@ function kube-up() {
     write-cluster-name
     create-autoscaler-config
     create-master
-    create-nodes-firewall
-    create-nodes-template
-    # Windows nodes take longer to boot and setup so create them first.
-    create-windows-nodes
-    create-linux-nodes
+    if [[ "${KUBE_CREATE_NODES}" == "true" ]]; then
```

Comment:
Do we really need that env var?
Can we maybe gate on NUM_NODES=0 instead?

Reply:
Added comments in config-default.sh and config-test.sh.